### PR TITLE
feat(runners): correctly paginate org runners and remove useless status label

### PR DESCRIPTION
**Pull request by jstewart612**

### CODE REVIEW

**Comment by jstewart612** [comment]
fixes #12

File: metrics/get_runners_organization_from_github.go [hunk 1]
```diff
@@ -18,42 +22,40 @@ var (
...
-			err = json.NewDecoder(resp.Body).Decode(&p)
-			if err != nil {
-				log.Fatal(err)
-			}
-			for _, r := range p.Runners {
-				if r.Status == "online" {
-					RunnersOrganizationGauge.WithLabelValues(orga, r.OS, r.Status, r.Name, strconv.Itoa(r.ID)).Set(1)
-				} else {
-					RunnersOrganizationGauge.WithLabelValues(orga, r.OS, r.Status, r.Name, strconv.Itoa(r.ID)).Set(0)
+			for {
```
Comment (Labbs) [inline metrics/get_runners_organization_from_github.go:43]: Indent

Comment (jstewart612) [reply metrics/get_runners_organization_from_github.go:43]: Unsure what you mean?

**Comment by Labbs** [comment]
All the changes were added in the code refactor. Available in v1.7.0

**Pull request closed without merging**